backend/src/common/util/workflow.go

```diff
@@ -161,10 +161,12 @@ func (w *Workflow) HasScheduledWorkflowAsParent() bool {
 	return containsScheduledWorkflow(w.Workflow.OwnerReferences)
 }
 
-func (w *Workflow) GetSpec() *Workflow {
-	spec := w.DeepCopy()
-	spec.Status = workflowapi.WorkflowStatus{}
-	return NewWorkflow(spec)
+func (w *Workflow) GetWorkflowSpec() *Workflow {
```

Comment:
You need to update the references to this in `resource_manager.go` and `resource_manager_test.go`